src/views/dashboard/playground/docs/getting-started.md

```diff
@@ -45,74 +61,142 @@ In GreptimeDB, there are three types of columns:
 - `TIME INDEX`: columns of data type TIME
 
 ```sql
-DESC TABLE system_metrics;
+DESC TABLE cpu_metrics;
 ```
 
 ## Add Some Data
 
-Using the `INSERT` statement to easily add data to the table. Below example inserts three rows into the `system_metrics` table.
+Using the `INSERT` statement to easily add data to the table. Below example inserts three rows into the `cpu_metrics` table.
+
+<!-- ts could be timestamp -->
 
 ``` sql
-INSERT INTO system_metrics
+INSERT INTO cpu_metrics
 VALUES
-    ("host1", "idc_a", 11.8, 10.3, 10.3, 1667446791450),
-    ("host2", "idc_a", 80.1, 70.3, 90.0, 1667446792460),
-    ("host1", "idc_b", 50.0, 66.7, 40.6, 1667446793470);
+    ('host_0','test',32,58,36,'2023-04-01T00:00:00+00:00'),
+    ('host_1','staging',12,32,50,'2023-04-01T00:00:00+00:00'),
+    ('host_2','test',98,5,40,'2023-04-01T00:00:00+00:00'),
+    ('host_3','test',98,95,7,'2023-04-01T00:00:00+00:00'),
+    ('host_4','test',32,44,11,'2023-04-01T00:00:00+00:00'),
+    ('host_0','test',31,57,37,'2023-04-01T00:01:00+00:00'),
+    ('host_1','staging',11,31,52,'2023-04-01T00:01:00+00:00'),
+    ('host_2','test',99,6,39,'2023-04-01T00:01:00+00:00'),
+    ('host_3','test',99,95,7,'2023-04-01T00:01:00+00:00'),
+    ('host_4','test',31,44,11,'2023-04-01T00:01:00+00:00'),
+    ('host_0','test',29,58,36,'2023-04-01T00:02:00+00:00'),
+    ('host_1','staging',10,32,50,'2023-04-01T00:02:00+00:00'),
+    ('host_2','test',100,5,36,'2023-04-01T00:02:00+00:00'),
+    ('host_3','staging',97,95,6,'2023-04-01T00:02:00+00:00'),
```

Comment:
```suggestion
    ('host_3','test',97,95,6,'2023-04-01T00:02:00+00:00'),
```

Reply:
Is it designed to have host_3 for both test and staging envs?

Reply:
yes, how do you know I changed the value 😂

Reply:
because host_3 has inconsistent environments...